src/test/regress/expected/upgrade_list_citus_objects.out

```diff
@@ -1,4 +1,8 @@
 -- list all postgres objects belonging to the citus extension
+--
```

Comment:
same here, I wouldn't mind filtering the any_value with a WHERE clause. This is a very rare case, and I'm fine to have less coverage (with the benefit of avoding multiple output files on these tricky tests)